### PR TITLE
MEPTS-1447 | Resumo Mensal PREP Diff ICAP DB

### PR DESCRIPTION
**Pull request by TinoMuianga**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/PrepNewCohortQueries.java [hunk 1]
```diff
@@ -56,7 +75,7 @@ public CohortDefinition getClientsWhoNewlyInitiatedPrep() {
             genericCohortQueries.getPatientAgeBasedOnPrepStartDate(15, 200),
             "startDate=${startDate},endDate=${endDate},location=${location}"));
```
Comment (helderjosue) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/PrepNewCohortQueries.java:76]: @TinoMuianga This fix is not correct! The Prep New Requirements, `PREP_NEW_FR2`  talks about **"clients included are the adults (>=15 years old) who initiated the treatment"** `PREP_NEW_FR4`, but RM PREP does not use the age verification. Please refer to the Prep New Requirements and apply a specific correction for this report

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/PrepNewCohortQueries.java [hunk 2]
```diff
@@ -56,7 +75,7 @@ public CohortDefinition getClientsWhoNewlyInitiatedPrep() {
...
 
-    cd.setCompositionString("(A AND C) AND NOT B");
+    cd.setCompositionString("A AND NOT B");
```
Comment (helderjosue) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/PrepNewCohortQueries.java]: @TinoMuianga This fix is not correct! The Prep New Requirements, `PREP_NEW_FR2`  talks about **"clients included are the adults (>=15 years old) who initiated the treatment"** `PREP_NEW_FR4`, but RM PREP does not use the age verification. Please refer to the Prep New Requirements and apply a specific correction for this report

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/PrepNewCohortQueries.java [hunk 3]
```diff
@@ -32,6 +32,25 @@ public PrepNewCohortQueries(
...
+   *         <li>“O(a) utente está a iniciar pela 1a vez a PrEP Data” registada na “Ficha de
+   *             Consulta Inicial PrEP”
+   *         <li>“Data de Início PrEP” registada na secção “Resumo da Ficha de Seguimento” da “Ficha
+   *             de Consulta Inicial PrEP”
+   *       </ul>
+   *   <li>Excluindo todos os utentes “Transferidos De” outra Unidade Sanitária (RF10)
+   * </ul>
+   *
+   * @return CohortDefinition
+   */
```
Comment (helderjosue) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/PrepNewCohortQueries.java]: @TinoMuianga please add the correct requirements for this method. Refer to Prep New requirements

**Review by helderjosue — CHANGES_REQUESTED** [review]
Please check the comments

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/PrepNewCohortQueries.java (hunk 2 above)
Comment (TinoMuianga) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/PrepNewCohortQueries.java]: Corrected.

**Review by helderjosue — APPROVED** [review]
LGTM

**Review by psmatsinhe — APPROVED** [review]
Looks good to me

**Pull request merged**